src/modules/Header/styles.css

```
@@ -1,35 +1,70 @@
-.baq-container {
-  padding-left: 20px;
-  padding-right: 20px;
-  background: #f2d901;
-  box-shadow: 3px 3px 15px rgba(0, 0, 0, 0.15);
+header {
+  display: grid;
 }
-.navbar-baq {
-  width: 100%;
-  display: flex;
-  justify-content: space-between;
-  max-width: 1152px;
-  margin: 0 auto;
+
+header .bottom {
+  display: none;
 }
-.navbar-baq .title {
-  font-size: 1.5rem;
-  color: black;
-}
-.icon-redes {
-  display: flex;
+
+header .top {
+  display: grid;
+  grid-template-columns: minmax(auto, .9fr) repeat(4, auto);
+  grid-gap: 10px;
+  justify-content: center;
   align-items: center;
-  justify-content: flex-end;
+  background-color: #F2D903;
+  box-shadow: 0 1px 10px 1px rgba(0, 0, 0, .3);
 }
-.icon-redes a {
-  color: #222;
-  padding: 7px 8px;
-  margin-left: 12px;
-  font-size: 24px;
+
+header .top img {
+  width: 40px;
+  height: 40px;
 }
-.navbar-baq .navbar-left-text {
-  display: flex;
+
+header .top div:first-of-type {
+  display: grid;
+  grid-template-columns: repeat(2, auto);
   align-items: center;
+  justify-content: start;
+  grid-gap: 0 10px;
+}
+
+header nav div i {
```

igual aquí, podrias agregar una clase y ya, para selcecionar el elemento ?
```suggestion
.element-name {

```

---

Si lo se, lo hago mas que todo por ahorrar variables y memoria.